_posts/0000-01-02-JTwallace.md

```diff
@@ -0,0 +1,5 @@
+layout: slide
+title: "Welcome to our second slide!"
+
+Your Text
+Use the left arrow to go back!
```

Comment:
All this _coding_ is making me hot time to remove some layers! :stuck_out_tongue_winking_eye: